plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfigurator.java

```diff
@@ -241,7 +242,11 @@ public CNode describe(T instance, ConfigurationContext context) throws Exception
             if (value != null) {
                 Object converted = Stapler.CONVERT_UTILS.convert(value, a.getType());
                 if (a.isMultiple()) {
-                    args[i] = Arrays.asList(converted);
+                    if (converted instanceof DescribableList) {
```

Comment:
I think `instanceof Describable` would be a better one and you could move it outside with an if, else if, else block :)

Reply:
I tried that but the test fails, didn't look too closely into why its not an instance of it, but you should be happier now with the new code